packages/frontend/lib/index.ts

```diff
@@ -119,9 +119,9 @@ export default class Nango {
         if (
             options &&
             'credentials' in options &&
-            (!('oauth_client_id_override' in options.credentials) || !('oauth_client_secret_override' in options.credentials)) &&
-            !('token_id' in options.credentials) &&
-            !('token_secret' in options.credentials) &&
+            (('token_id' in options.credentials && 'token_secret' in options.credentials) ||
+                !('oauth_client_id_override' in options.credentials) ||
+                !('oauth_client_secret_override' in options.credentials)) &&
```

Comment:
do we still need the following block below if we are now getting into `customAuth`:

```
        if (options && 'credentials' in options && 'token_id' in options.credentials && 'token_secret' in options.credentials) {
            url = this.hostBaseUrl + `/auth/tba/${providerConfigKey}${this.toQueryString(connectionId, options as ConnectionConfig)}`;
        }
```

Reply:
Thanks, removed